src/services/post-services.ts

```diff
@@ -0,0 +1,35 @@
+import axios from "axios";
+const SERVER_API_URL = process.env.REACT_APP_SERVER_API_URL;
+const POSTS_URL = `${SERVER_API_URL}/posts`;
+
+const api = axios.create({ withCredentials: true });
+
+export const createPostWithRecipe = async ({post, recipe}) => {
+
+  console.log(post);
+  console.log(recipe);
+
+  try {
+    const response = await api.post(`${POSTS_URL}`, {
+      post,
+      recipe
+    });
+    post = response.data;
```

Comment:
nit: combine line 17 and 18 to `return response.data`?

Reply:
was for readability to show what the response is, but that can just be documented with a comment